Formula/luajit.rb

```diff
@@ -1,21 +1,38 @@
+# NOTE: We have a policy of building only from tagged commits, but make a
+#       singular exception for luajit. This exception will not be extended
+#       to other formulae. See:
+#       https://github.com/Homebrew/homebrew-core/pull/99580
 class Luajit < Formula
   desc "Just-In-Time Compiler (JIT) for the Lua programming language"
   homepage "https://luajit.org/luajit.html"
+  # Update this to the tip of the `v2.1` branch at the start of every month.
```

Comment:
Maybe a link to where you'd find the commit hash for this branch would be helpful?

Reply:
Yeh, the first feels like it'd be a good add as a comment!